goes-plume-viewer/src/utils/trieTree.js

```diff
@@ -21,7 +21,11 @@ export class TrieTree{
             node = node.children[letter]
         }
         node.isCompleteWord = true;
-        node.word = word;
+        if (alternateWord) {
+            node.word = alternateWord;
+        } else {
+            node.word = word;
+        }
```

Comment:
👀

Reply:
a little enhancement on typical trie tree. i.e. for each word in the plume Id, we should show the result for the plumeId and not that word completion. for instance, Typing in `Duran` (for Durango) in search should recommend Mexico_Durango_BV1_BV1-1 and not just Durango.